db/queries.py

```diff
@@ -3,44 +3,75 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    mes_list = list(Message.objects.filter(text__icontains=word))
+    return mes_list
```

Comment:
no need to create variable 
fix here and other places 
```suggestion
    return list(Message.objects.filter(text__icontains=word))
```